appveyor.yml

```diff
@@ -215,7 +215,7 @@ test_script:
       }
   - cd electron
   - if "%RUN_TESTS%"=="true" ( echo Running main test suite & node script/yarn test -- --trace-uncaught --runners=main --enable-logging=file --log-file=%cd%\electron.log )
-  - if "%RUN_TESTS%"=="true" ( echo Running remote test suite & node script/yarn test -- --trace-uncaught --runners=remote --runTestFilesSeperately --enable-logging=file --log-file=%cd%\electron.log )
+  - if "%RUN_TESTS%"=="true" ( echo Running remote test suite & node script/yarn test -- --trace-uncaught --runners=remote --runTestFilesSeparately --enable-logging=file --log-file=%cd%\electron.log )
```

Comment:
It's possible this change (implementation below) could be deemed an "API" change, but I don't think anyone outside electron/electron cares.

Reply:
That flag is used internally for Electron to run our tests, so this wouldn't be an API change. As long as the flag is changed in every location where its used (which seems to be the case here), then I'm fine with this change. The only issue would be if changing it broke the usage of the flag.